torchrl/collectors/llm_collector.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from tensordict import lazy_stack, TensorDictBase
+
+from torchrl.collectors import (
+    LocalWeightUpdaterBase,
+    RemoteWeightUpdaterBase,
+    SyncDataCollector,
+)
+from torchrl.data import ReplayBuffer
+
+
+class LLMCollector(SyncDataCollector):
+    """A simplified version of SyncDataCollector for LLM inference."""
```

Comment:
missing kwarg description + running example (feel free to pick the one in the tests)